.circleci/config.yml

```diff
@@ -151,9 +151,9 @@ commands:
           command: |
             export LOCAL_BASE=~/.local-${OS_NAME}
             .circleci/fetch_geth_parity_solc.sh
-            ln -s ${LOCAL_BASE} ~/.local
+            ln -sfn ${LOCAL_BASE} ~/.local
       - save_cache:
-          key: system-deps-v3-{{ arch }}-geth-{{ checksum "/tmp/geth-version" }}-parity-{{ checksum "/tmp/parity-version" }}-solc-{{ checksum "/tmp/solc-version" }}
+          key: system-deps-v4-{{ arch }}-geth-{{ checksum "/tmp/geth-version" }}-parity-{{ checksum "/tmp/parity-version" }}-solc-{{ checksum "/tmp/solc-version" }}
```

Comment:
what is the version number for here? This is how you wipe the cache? IF yes, isn't there any way from inside CircleCI?

Reply:
Yes basically. It's not possible to wipe the cache, just force using a new one by changing the cache key. The version number approach is the one they suggest in their docs.